src/main/java/org/tron/common/runtime/vm/VM.java

```diff
@@ -1223,7 +1226,7 @@ public void step(Program program) {
         }
         break;
         case RETURN:
-        case REVERT: {
+        case REVERT:
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Reduce this switch case number of lines from 15 to at most 5, for example by extracting code into methods. ([squid:S1151](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1151))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1681&issues=AWbnVGTXLpqqSUlc5prB&open=AWbnVGTXLpqqSUlc5prB)

[ISSUE_KEY]: AWbnVGTXLpqqSUlc5prB